Libraries/zephyr/MAX/Source/MAX78000/max78xxx_system.c

```diff
@@ -0,0 +1,24 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2024 Analog Devices, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+#include "max78000.h"
+
+/* 
+ * This function is called during boot up.
+ */
+void max32xx_system_init(void) {}
```

Comment:
As I see, "MXC_ICC_Enable(MXC_ICC0)" line required in here. Please check [system_max78000.c](https://github.com/analogdevicesinc/msdk/blob/main/Libraries/CMSIS/Device/Maxim/MAX78000/Source/system_max78000.c#L134) file.